packages/api/src/command/medical/document/document-download.ts

```diff
@@ -138,3 +146,23 @@ export async function getSignedURL({
 
   return s3Utils.getSignedUrl({ bucketName: bucket, fileName });
 }
+
+/**
+ * Gets the download URL for the raw hl7 message.
+ * @param fileName - the name of the file in the bucket
+ * @returns the download URL for the raw hl7 message
+ */
+export async function getIncomingHl7MessageDownloadUrl({
```

Comment:
this looks like HL7-specific stuff, so we should prob move this to that specific folder?